pieforproviders/pieforproviders
# Feature/195/resend verification email

# 💅🏼 What issue does this fix?
#195 - users can resend the verification email after signing up

Also adds e2e tests for Signup

## 🍂 Before You Submit


## 🧵 Steps to set up locally

You'll need to restart your yarn server or rebuild tailwind manually because of a very minor config change

## 🧳 Steps to test
- visit `/signup`
- Sign up for an account
- you should be redirected to the Confirmation Sent page
- you should receive a confirmation email
- click the "Click here to resend the confirmation email" link
- you should see an "Email resent!" success box (per Slack discussions w/ @rebeccakarasiktw this is how we want to display these because the message object in the Figma mockup isn't positionable; it's always at the top center, instead of under the resend link)
- you should receive another confirmation email

## 🕯 Caveats, concerns
We might eventually want to put this ConfirmationSent view with the "resend" link in another view that someone can get redirected to if they're trying to log in before confirming. @csprayregen we'll need a ticket for that.
